cassandra/src/main/java/com/yahoo/ycsb/db/CassandraCQLClient.java

```diff
@@ -171,9 +174,17 @@ public void init() throws DBException {
             getProperties().getProperty(WRITE_CONSISTENCY_LEVEL_PROPERTY,
                 WRITE_CONSISTENCY_LEVEL_PROPERTY_DEFAULT));
 
+        Boolean useSSL = Boolean.parseBoolean(getProperties().getProperty(USE_SSL_CONNECTION,
+            DEFAULT_USE_SSL_CONNECTION));
+
         if ((username != null) && !username.isEmpty()) {
-          cluster = Cluster.builder().withCredentials(username, password)
-              .withPort(Integer.valueOf(port)).addContactPoints(hosts).build();
+          if (useSSL) {
```

Comment:
Can these lines of code be merged into just a couple of lines like below?

clusterBuilder = Cluster.builder().with.....addContactPoints(hosts)
if (useSSL) ClusterBuilder = clusterBulider.withSSL()
Cluster = clusterBuilder.build()

Reply:
Sure let me try and optimize it.